cypress/e2e/wallet-connection/connect.test.ts

```diff
@@ -0,0 +1,38 @@
+import { getTestSelector } from '../../utils'
+
+describe('disconnect wallet', () => {
+  it('should clear state', () => {
+    cy.visit('/swap', { ethereum: 'hardhat' })
+    cy.get('#swap-currency-input .token-amount-input').clear().type('1')
+
+    // Verify wallet is connected
+    cy.hardhat().then((hardhat) => cy.contains(hardhat.wallet.address.substring(0, 6)))
```

Comment:
you might consider saving `hardhat.wallet.address.substring(0, 6)` as a var for less repetition

Reply:
are you looking for the account address that exists in the web3status component here? 🤔 maybe use `shortenAddress` since it's used in that component?